apps/web/src/pages/Pools/CreatePoolPage/CreatePoolPage.tsx

```diff
@@ -60,7 +60,6 @@ const CreatePoolCommunity_Fragment = graphql(/* GraphQL */ `
       en
       fr
     }
-    teamIdForRoleAssignment
```

Comment:
Why was this removed?  It's still in the schema.

Reply:
It was no being used anymore in this instance as `teamFromCommunity` was also removed here as we no longer need to derive the team from the community for a pool.

Reply:
Yeah, seems good.  Pool creation still seem to be working normally.